docker-cloud/builds/automated-build.md

```diff
@@ -105,6 +105,21 @@ run an initial test.
     Docker Cloud on every push. Only pushes to branches that are listed as the
     source for one or more tags will trigger a build.
 
+### Environment variables for builds
+
+You can set environment variables used in your build either by declaring them in
```

Comment:
"by declaring them in hooks files" --> we cannot say this. the message should be that the build env vars defined in the UI are accessible from your hooks. other approaches are just wrong and cannot be commented or proposed as alternatives.
If you want to dig in this idea, make a use case for env vars, tell that you need env vars in your hooks to cover this use case, say that declaring them in your git repo is unacceptable due to security concerns, and then conclude that defining them in the Docker Cloud UI is the right solution.